change/change-1a2fe4f0-bbf0-4942-8775-8667fced9a81.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "type": "patch",
+      "comment": "High contrast style fixes for ColorPicker",
+      "packageName": "@fluentui/react",
+      "email": "sarah.higley@microsoft.com",
+      "dependentChangeType": "patch"
+    }
+  ]
+}
```

This needs to be un-grouped for now
```suggestion
{
  "type": "patch",
  "comment": "High contrast style fixes for ColorPicker",
  "packageName": "@fluentui/react",
  "email": "sarah.higley@microsoft.com",
  "dependentChangeType": "patch"
}
```